lte/gateway/python/magma/pipelined/openflow/flows.py

```diff
@@ -144,7 +144,7 @@ def add_flow(datapath, table, match, actions=None, instructions=None,
     # As 4G GTP tunnel, Register value is not used.
     # if use for default flow, 4G traffic is dropping.
     if ((goto_table != CLASSIFIER_NEXT_TABLE_NUM) and
-                     (goto_table != CLASSIFIER_INTERNAL_SAMPLING_FWD_TABLE_NUM)):
+            (goto_table != CLASSIFIER_INTERNAL_SAMPLING_FWD_TABLE_NUM)):
```

Comment:
Also I don't understand the purpose of this `if ((goto_table != CLASSIFIER_NEXT_TABLE_NUM) and` either, why are we hard coding a specific table value here. 
We already have add_resubmit_next_service_flow function, lets use that instead of add_flow. Sorry I missed that in the previous reviews.

Lets just add a flag reset_default_register to add_resubmit_next_service_flow and remove this goto_table comparison.